data/hai/hai reveal 4 middle.txt

```diff
@@ -137,7 +136,7 @@ mission "Hai Reveal [A02] Symbol Identification"
 			choice
 				`	"Yes, I have."`
 				`	"No, I've never seen it before."`
-			`	She nods. "This symbol has been found a number of times in association with the circumstances where rescued Hai have had some level of cadmium poisoning. We have not been able to identify where it originates from. We must know and seek out who they are and where they are hiding, though I believe our mutual friend will be satisfied with merely the 'who' for now."`
+			`	Wayou nods. "This symbol has been found a number of times where rescued Hai have had of cadmium poisoning, but we cannot identify its origin or meaning. This may be key to finding the kidnappers' identity and whereabouts."`
```

Comment:
```suggestion
			`	Wayou nods. "This symbol has been found a number of times where rescued Hai have had cadmium poisoning, but we cannot identify its origin or meaning. This may be key to finding the kidnappers' identity and whereabouts."`
```